### ISSUE TITLE
Problems when fine-tune BEiT3 on VQA

### ISSUE THREAD

**Issue by Alchemistyui** [microsoft/unilm#1099]
Thank you for your excellent work! 

I cannot reproduce your result when I'm fine-tuning  BEiT3 following your guidance [get_started_for_vqav2.md](https://github.com/microsoft/unilm/blob/456511b24f5fb84bae8b5e61bba810e7b864e411/beit3/get_started/get_started_for_vqav2.md). I'd appreciate it if you can give me some suggestions.

**Describe**
I'm using 8 40G-A100 for fine-tuning, and not using docker but using conda environment. The other setups are the same as the guidance.

I've tried to train BEiT-3 base and BEiT-3 large with the script you provided, but there is still a big gap between my implementation performance and yours, as shown in the table:

| Dataset        | Model       | Official Result | My result |
| -------------- | ----------- | --------------- | --------- |
| VQAv2 val.     | beit3_base  | 86.1%           | 74.40%    |
| VQAv2 test-dev | beit3_base  | 77.65%          | 73.1%     |
| VQAv2 test-dev | beit3_large | 81.85%          | 68.8%     |

The train loss and accuracy looks fine (as follows), and I also upload the log, conda list and tensorboard file here: [beit3 finetune log](https://drive.google.com/drive/folders/1jBgqnqgLB8CmgZGVnT4AHBp2MFQ-U0Uz?usp=sharing).

![tensorboard curve](https://drive.google.com/u/2/uc?id=1fRZrd-dKPRnjYRblMJ-HtfkNf5Sd6_gH&export=download)

I'd appreciate it if you can help me. Thank you very much!

**Comment by ShiyuZhangUST** [comment]
May I ask how did you download beit-v3 pre-trained weights?
I got PublicAccessNotPermitted error when trying to download beit-v3 pre-trained weights from https://conversationhub.blob.core.windows.net/beit-share-public/beit3/vqa/beit3_base_patch16_480_vqa.pth

> <Error>
<Code>PublicAccessNotPermitted</Code>
<Message>Public access is not permitted on this storage account. RequestId:c41c88a3-901e-0059-75cf-997f8e000000 Time:2023-06-08T06:06:09.4120160Z</Message>
</Error>

**Comment by Alchemistyui** [comment]
> May I ask how did you download beit-v3 pre-trained weights? I got PublicAccessNotPermitted error when trying to download beit-v3 pre-trained weights from https://conversationhub.blob.core.windows.net/beit-share-public/beit3/vqa/beit3_base_patch16_480_vqa.pth
> 
> `<Error> <Code>PublicAccessNotPermitted</Code> <Message>Public access is not permitted on this storage account. RequestId:c41c88a3-901e-0059-75cf-997f8e000000 Time:2023-06-08T06:06:09.4120160Z</Message> </Error>`

**Comment by Alchemistyui** [comment]
> May I ask how did you download beit-v3 pre-trained weights? I got PublicAccessNotPermitted error when trying to download beit-v3 pre-trained weights from https://conversationhub.blob.core.windows.net/beit-share-public/beit3/vqa/beit3_base_patch16_480_vqa.pth
> 
> `<Error> <Code>PublicAccessNotPermitted</Code> <Message>Public access is not permitted on this storage account. RequestId:c41c88a3-901e-0059-75cf-997f8e000000 Time:2023-06-08T06:06:09.4120160Z</Message> </Error>`

I download the pretrained weight according to their table here: https://github.com/microsoft/unilm/tree/master/beit3#fine-tuning-on-vqav2-visual-question-answering.

For example, [beit3_base_patch16_224](https://conversationhub.blob.core.windows.net/beit-share-public/beit3/pretraining/beit3_base_patch16_224.pth?sv=2021-10-04&st=2023-06-08T11%3A16%3A02Z&se=2033-06-09T11%3A16%3A00Z&sr=c&sp=r&sig=N4pfCVmSeq4L4tS8QbrFVsX6f6q844eft8xSuXdxU48%3D).
